src/main/java/com/volasoftware/tinder/service/EmailSenderServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.volasoftware.tinder.service;
+
+import com.volasoftware.tinder.model.User;
+import com.volasoftware.tinder.model.Verification;
+import jakarta.mail.MessagingException;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+@Service
+public class EmailSenderServiceImpl implements EmailSenderService {
```

Comment:
This service class should only send emails to some email/s with some subject and some content. Please, remove all usages with not related objects from this class as tokens, users, files and etc.